uberenv.py

```diff
@@ -233,28 +240,42 @@ def __init__(self, opts, extra_opts):
 
         # load project settings
         self.project_opts = load_json_file(opts["project_json"])
-        print("[uberenv project settings: {}]".format(str(self.project_opts)))
-        print("[uberenv options: {}]".format(str(self.opts)))
+        print("[uberenv project settings: {0}]".format(str(self.project_opts)))
+        print("[uberenv options: {0}]".format(str(self.opts)))
+
+        # setup main package name
+        self.pkg_name = self.project_opts["package_name"]
 
     def setup_paths_and_dirs(self):
-        self.uberenv_path = os.path.dirname(os.path.realpath(__file__))
+        self.uberenv_path = os.path.split(os.path.abspath(__file__))[0]
```

Comment:
I’m interested in the explanation behind this change. Was it causing an issue on Windows?
(I can’t remember why I chose realpath here, but I remember using it deliberately, should have commented...)

Reply:
I am not sure that it was intentional.
Crawling through the git history, it might have been done by git automerge?

I'll change it back, but I'm curious about the difference.

It seems to me like your change would be better able to handle things like windows directories with spaces in them (?)